qrand.go

```diff
@@ -66,6 +68,25 @@ func Read(buf []byte) (n int, err error) {
 	return Reader.Read(buf)
 }
 
+type ΨSource struct{}
+
+var qSource ΨSource
```

Comment:
We don't need this global variable here, since `Source` is exported, so people can create their own instances of it if they want to.